macros/drop_schema.sql

```diff
@@ -0,0 +1,20 @@
+{% macro drop_schema(schema_name) %}
+
+    {#/* Drops schema named as `schema_name` and all its objects.
+       ARGS:
+         - schema_name (string) : name of schema that will be dropped.
+       RETURNS: nothing to the call point.
+       SUPPORTS:
+            - Postgres
+            - Snowflake
+    */#}
+
+  {% set sql %}
+      DROP SCHEMA IF EXISTS {{schema_name}} CASCADE;
```

Comment:
At the very beginning I thought that it would be useful to apply [adapter.drop_schema()](https://docs.getdbt.com/reference/dbt-jinja-functions/adapter#drop_schema) macro here but faced with errors trying to run command
`dbt run-operation drop_schema --profiles-dir . --args '{schema_name: FEATUREDATA3408_TEST}' --target snowflake`
that runs perfectly in the current implementation variant which is more clear and workable.